### DISCUSSION TITLE
Client side caching not working after re-connect

### DISCUSSION THREAD

**Discussion by rreddy13Official** [redis/lettuce#1728] [Q&A]
Hi , 

I am using lettuce(6.0.1.RELEASE) to connect to Redis 6.0.9 and enabled client side caching. Everything works fine as long as the connection is active. When connection is interrupted due to network, it tries to reconnect. After successful reconnection, Client side caching is not applied to the new connection. 

 **How could I solve this so that the client side caching works fine even after reconnect ?** 

I tried to enable the client side caching though RedisConnectionStateListener.onRedisConnected() but its not working(give the timeout exception). Please advice.

**Comment by rreddy13Official** [comment]
Got the answer in gitter:
Lettuce provides a frontend with cache accessors to wrap an existing cache and augment it with Redis functionality. I suggest flushing the underlying client-side cache to work around that problem. You can either register connection state listeners via RedisClient or listen to the EventBus for disconnect events to get notified of disconnects.

**Reply by marchpane** [reply]
Hi @rreddy13Official,
I know it's an old post from you but may I ask if you happen to know how a Spring Boot application can receive/listen for events from the EventBus of Lettuce?
